Makefile

```
@@ -156,7 +156,7 @@ docker-image: ## Build docker image
 validate-runtime-images: ## Validates delivered runtime-images meet minimum criteria
 	@required_commands=$(REQUIRED_RUNTIME_IMAGE_COMMANDS) ; \
 	for file in `find etc/config/metadata/runtime-images -name "*.json"` ; do \
-		image=`grep image_name $$file | awk '{print $$2}' | sed s/\"//g` ; \
+		image=`grep image_name $$file | awk '{print $$2}' | sed s/\"//g | sed s/,//g` ; \
```

We can combine the sed statements with the following...
```suggestion
		image=`grep image_name $$file | awk '{print $$2}' | sed s/[\",]//g` ; \
```

---

The test is now passing with the updated change